.github/workflows/arm-ttk.yml

```diff
@@ -27,6 +27,17 @@ jobs:
     runs-on: ubuntu-latest
     
     steps:
+      - name: Apply arm-templates label on PR
+        uses: actions/github-script@v6
+        if: ${{ github.event_name == 'pull_request' }}
+        with:
+          script: |
+            github.rest.issues.addLabels({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              labels: ['arm-templates']
+            })
```

Comment:
Hi, this add the ``arm-templates`` label to the issue, but not to the Pull request ?

Reply:
It adds the label only on the pull request triggering the ci job